test/e2e/webdriver/firefox.js

```diff
@@ -19,11 +19,25 @@ const { retry } = require('../../../development/lib/retry');
 const TEMP_PROFILE_PATH_PREFIX = path.join(os.tmpdir(), 'MetaMask-Fx-Profile');
 
 /**
- * Proxy host to use for HTTPS requests
+ * Default proxy host to use for HTTPS requests
+ *
+ * @type {string}
  */
-const HTTPS_PROXY_HOST = new URL(
-  process.env.SELENIUM_HTTPS_PROXY || 'http://127.0.0.1:8000',
-);
+const DEFAULT_PROXY_HOST = 'http://127.0.0.1:8000';
```

Comment:
se chrome comment about scope

Reply:
thank you that's a great suggestion :heart_eyes: !! I've implemented the changes